crates/proto/src/rr/record_type.rs

```diff
@@ -181,6 +184,8 @@ impl From<u16> for RecordType {
         match value {
             1 => RecordType::A,
             28 => RecordType::AAAA,
+            // FIXME: wrong value here
+            2005 => RecordType::ANAME,
```

Comment:
Random comment from a DNS-nerd: Please use a number in the "private" range (65280-65534) for types not yet [allocated by IANA](https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-4).

Reply:
I second this.

Reply:
Will do. I was hoping there was an official value I hadn’t discovered yet.

Reply:
I opened https://github.com/each/draft-aname/issues/48 to work on the RR Type application.

Reply:
Added a tracking issue: #723